cloudpunch/ostlib/oscompute.py

```diff
@@ -214,8 +214,9 @@ def create(self, instance_name, image_id, flavor_name, network_id,
                                                      security_groups=[secgroup_id])
         logging.debug('Created instance %s with ID %s using image %s and flavor %s',
                       instance_name, self.get_id(), image_id, flavor_name)
-        logging.debug('Waiting for instance %s with ID %s to finish building',
-                      instance_name, self.get_id())
+        waitTime=2*retry_count
```

Comment:
flake8 requires white space around the =

Reply:
Code is now flake8 compliant.